### ISSUE TITLE
Feature: Settings button move

### ISSUE THREAD

**Issue by theotherlinh** [pixelfed/pixelfed#3841]
Getting to: https://pixelfed.social/settings/home 
is a bit annoying. You have to go to your profile, then find the gear icon.  

I would suggestion this should all just be under your avatar at the top right. Quicker and it just makes more sense to me given that's where I would think most people expect settings to be. It'd help in trying to explain where to check your settings for new folks, too.

**Comment by mitexleo** [comment]
Yeah ... It's Necessary for new users

**Comment by nemobis** [comment]
Agreed this is confusing in the "new UI" and inconsistent. (Tested in v0.11.13.)

From the main UI, there are two dropdowns for user options: one in the top bar and one from the user's profile's avatar. The top bar doesn't link the settings (/settings/home) but only "UI settings", which don't include all UI-related settings. The other one does link settings. Then, when you reach the settings page, the top bar changes and the settings are included from the dropdown.

The main UI:

![Screenshot_20240414_213504](https://github.com/pixelfed/pixelfed/assets/901528/3570f91a-68e2-459b-bb61-ddf9d95297a7)
![Screenshot_20240414_213516](https://github.com/pixelfed/pixelfed/assets/901528/dcfa39ba-c0f9-4ee1-9d67-c67fb8e76a15)

The UI in settings page:

![Screenshot_20240414_213410](https://github.com/pixelfed/pixelfed/assets/901528/17a81a0a-2fb4-415f-89ed-184cce679b09)
